cmake/CMakeGraphVizOptions.cmake

```diff
@@ -0,0 +1,5 @@
+set(GRAPHVIZ_GRAPH_NAME "Pharo VM Dependencies")
```

Comment:
Maybe we do not want spaces here? Because this is going to be used above as file name! And spaces in file names are always a source of trouble...